compiler/rustc_codegen_llvm/src/llvm/ffi.rs

```diff
@@ -1134,11 +1134,7 @@ extern "C" {
     pub fn LLVMConstIntToPtr<'a>(ConstantVal: &'a Value, ToType: &'a Type) -> &'a Value;
     pub fn LLVMConstBitCast<'a>(ConstantVal: &'a Value, ToType: &'a Type) -> &'a Value;
     pub fn LLVMConstPointerCast<'a>(ConstantVal: &'a Value, ToType: &'a Type) -> &'a Value;
-    pub fn LLVMConstExtractValue(
-        AggConstant: &Value,
-        IdxList: *const c_uint,
-        NumIdx: c_uint,
-    ) -> &Value;
+    pub fn LLVMRustGetAggregateElement(ConstantVal: &Value, Idx: c_uint) -> &Value;
```

Comment:
```suggestion
    pub fn LLVMRustGetAggregateElement(ConstantVal: &Value, Idx: c_uint) -> Option<&Value>;
```
As this API can return null.